alerting_alert_rule_test.go

```diff
@@ -22,6 +22,27 @@ func TestAlertRules(t *testing.T) {
 		}
 	})
 
+	t.Run("get alert rule group succeeds", func(t *testing.T) {
+		server, client := gapiTestTools(t, 200, getAlertRuleGroupJSON)
+		defer server.Close()
+
+		group, err := client.AlertRuleGroup("d8-gk06nz", "test")
+
+		if err != nil {
+			t.Error(err)
+		}
+		t.Log(pretty.PrettyFormat(group))
```

Comment:
Do we actually need this to be printed or was this accidentally left here?

Reply:
Good catch on the other prints though, they show the same data as this and so the info is duplicated. I'll address the other comments but keep this one.